client/src/lua/config.rs

```diff
@@ -15,10 +16,10 @@ const INIT_FILE_FALLBACK_PATH: &'static str = "/etc/way-cooler/rc.lua";
 
 pub const DEFAULT_CONFIG: &'static str = include_str!("../../../config/rc.lua");
 
-/// Finds the configuration file and returns (filename, content)
+/// Reads the init file and returns it's contents
 pub fn load_config(lua: Context, cmdline_config: Option<&str>) -> io::Result<(String, String)> {
-    let (init_dir, mut init_file) = get_config(cmdline_config)?;
-    if init_dir.components().next().is_some() {
+    let (init_path, mut init_file) = get_config(cmdline_config)?;
+    if let Some(init_dir) = init_path.parent() {
```

Comment:
I could not figure out what the previous if-condition was doing here

Reply:
err...yeah I'm not sure what that was doing.

This looks more correct :+1: